packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -173,31 +240,32 @@ export function DesktopSession(props: State) {
   }
 
   if (processing) {
+    // We don't know whether another session for this desktop is active while the
+    // fetchAttempt is still processing, so hold off on starting a TDP connection
+    // until that information is available.
+    const initTdpCli = fetchAttempt.status !== 'processing';
+
     return (
-      <Session {...props}>
+      <Session initTdpCli={initTdpCli} displayCanvas={false} {...props}>
```

Comment:
opt nit, just makes it look like we may overwrite props
```suggestion
      <Session {...props} initTdpCli={initTdpCli} displayCanvas={false}>
```